client/extensions/woocommerce/app/dashboard/manage-no-orders-view.js

```diff
@@ -99,6 +100,7 @@ class ManageNoOrdersView extends Component {
 	render = () => {
 		return (
 			<div className="dashboard__manage-no-orders">
+				<LabelsSetupNotice />
```

Comment:
Does it make sense in this page though? The user haven't even received any orders yet at this point.

Reply:
+1. This should only be added to the dashboard with orders present.